src/Consolonia.NUnit/README.md

```
@@ -0,0 +1,14 @@
+# Consolonia UI
+
+TUI (Text User Interface) (GUI Framework) implementation for [Avalonia UI](https://github.com/AvaloniaUI)
+
+Supports XAML, data bindings, animation, styling and the rest from Avalonia.
+
+> Project is in proof of concept state and is looking for collaboration.
+
+## Showcase (click picture to see video)
+[![datagridpic](https://user-images.githubusercontent.com/10516222/141980173-4eb4057a-6996-45bf-83f6-931316c98d88.png)](https://youtu.be/ttgZmbruk3Y)
+
+Example of usage: https://github.com/jinek/ToolUI
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)